syntax = "proto3";

package profiles.v1;

import "google/protobuf/timestamp.proto";

message UserProfile {
  string id = 1;
  string name = 2;
  string email = 3;
  google.protobuf.Timestamp last_seen = 4;
  string subscription_tier = 5;
  map<string, string> preferences = 6;
}
